corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -30,7 +30,7 @@ def _should_sync(restore_state):
 
     sync_interval = restore_state.restore_user.get_mobile_ucr_sync_interval()
     return not last_sync_log or (
```

Comment:
not blocking, but now that everything uses ors it might read easier if `not last_sync_log` was within the parentheses